api/routes/assessment_routes.py

```diff
@@ -141,10 +142,13 @@ def assessment_stats_for_fund_round_id(
     assessments = get_metadata_for_fund_round_id(
         fund_id=fund_id, round_id=round_id
     )
+    assessment_ids = [application['application_id'] for application in assessments ]
+
     qa_completed_assessments = [
         flag["application_id"]
-        for flag in get_latest_flags_for_each("QA_COMPLETED")
+        for flag in retrieve_flags_for_applications(assessment_ids)
```

Comment:
Don't think this is actually filtering on `QA_COMPLETED` anymore. it would just return all flags?

Reply:
It filters later on however, have changed to use a query just looking for applications of QA_COMPLETED now too